drivers/hud_twenty_twenty_two_to_twenty_twenty_four/config/initializers/hud_twenty_twenty_two_to_twenty_twenty_four_feature.rb

```diff
@@ -22,4 +22,6 @@
   ].each do |version|
     Importers::HmisAutoMigrate.add_migration(version, 'HudTwentyTwentyTwoToTwentyTwentyFour::CsvTransformer')
   end
+
+  Rails.application.config.queued_tasks[:hud_twenty_twenty_two_to_twenty_twenty_four_up] = -> { HudTwentyTwentyTwoToTwentyTwentyFour::DbTransformer.up }
```

Comment:
Yeah, so this is just letting the app know about a "one-time" task that should be run after deployment.  The TaskQueue model loops over these and runs them if they haven't been run bevore.  It's in the initalizer because we didn't want to put it in a migration that would get stomped on the next time we recreate a clean struture file.

Reply:
Yeah, so this is just letting the app know about a "one-time" task that should be run after deployment. The TaskQueue model loops over these and runs them if they haven't been run bevore. It's in the initalizer because we didn't want to put it in a migration that would get stomped on the next time we recreate a clean struture file.